### DISCUSSION TITLE
LSP Release

### DISCUSSION THREAD

**Discussion by agg23** [KarolS/millfork#105] [Proposals for 0.4]
My LSP PR #81 still needs a decent bit of work, but it also offers a large value add (at least to me). Would we be interested in polishing it for the 0.4 release?

**Comment by KarolS** [comment]
I'm even thinking about integrating it sooner, maybe in the next or the second-next 0.3.x release, as the most long-standing important issues have been fixed and I can focus on the more quality-of-life stuff.

Current plans involve few small optimization ideas (both for the generated code and for the compiler itself) and annotations/hints, so nothing very major. I think I can take a look at LSP as well.

Between 0.3.24 and 0.3.26 I was very busy with personal matters and also I had a hard drive crash, so it took me a while to get back to speed. But now it's all good.

**Reply by agg23** [reply]
Ok, that sounds great! I have a number of improvements queued after my recent NES game release for a contest, so I'm hoping to sit down and send those improvements your way in the next few days (both compiler fixes/bugs and LSP tweaks).
